.circleci/config.yml

```diff
@@ -99,8 +99,7 @@ jobs:
       - *run_yarn
       - run:
           environment:
-            RELEASE_CHANNEL: stable
-          command: yarn test --maxWorkers=2
+          command: yarn test --release-channel=stable --ci
```

Comment:
nit: I think `environment` can be removed now that it's empty, and `command` can be collapsed as well so this is just
```yaml
run: yarn test --release-channel=stable --ci
```